src/main/java/seedu/address/logic/commands/delivery/DeliveryCreateNoteCommand.java

```diff
@@ -0,0 +1,117 @@
+package seedu.address.logic.commands.delivery;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NOTE;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_DELIVERIES;
+
+import java.util.Optional;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.delivery.Delivery;
+import seedu.address.model.delivery.DeliveryDate;
+import seedu.address.model.delivery.DeliveryName;
+import seedu.address.model.delivery.DeliveryStatus;
+import seedu.address.model.delivery.Note;
+import seedu.address.model.delivery.OrderDate;
+import seedu.address.model.person.Customer;
+
+/**
+ * Represents a Command to create a note for a delivery
+ */
+public class DeliveryCreateNoteCommand extends Command {
+
+    public static final String COMMAND_WORD = DeliveryCommand.COMMAND_WORD + " " + "note";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a note to the delivery identified "
+        + "by the ID of the delivery. Existing note if any will be replaced with the input note.\n"
+        + "Parameters: DELIVERY_ID (must be a integer representing a valid ID) "
+        + PREFIX_NOTE + "NAME\n"
+        + "Example: " + COMMAND_WORD + "1 --note This is a note";
```

Comment:
Would a whitespace before 1 in the string, and using the static variable `PREFIX_NOTE` be better?

Reply:
Done, thanks for the feedback